licenses.go

```diff
@@ -41,43 +41,42 @@ func SetFirstEnterpriseCommit(sha string) {
 	firstEnterpriseCommit = sha
 }
 
-func CheckMenderCompliance(t *testing.T) error {
+func CheckMenderCompliance(t *testing.T) {
```

Comment:
Good! But you need to fix [the tests](https://github.com/mendersoftware/mendertesting/blob/master/license_test.go#L53).